Feature/geo reports (#39)
fivetran/dbt_pinterest: README.md

    pinterest_schema: your_schema_name 
```

### Step 4: Enable/disable models and sources
This package takes into consideration that not every Pinterest account tracks `keyword` performance, and allows you to disable the corresponding functionality by adding the following variable configuration:

```yml
vars:
    pinterest__using_keywords: False # Default = true
```

Additionally, your Pinterest Ads connection may not sync every table that this package expects. If you do not have the `PIN_PROMOTION_TARGETING_REPORT`, `TARGETING_GEO`, or `TARGETING_GEO_REGION` tables synced, add the following variable to your root `dbt_project.yml` file:

```yml
vars:

>     pinterest__using_pin_promotion_targeting_report: false # Default is true
>     pinterest__using_targeting_geo: false # Default is true, will be overridden if pinterest__using_pin_promotion_targeting_report is set to False
>     pinterest__using_targeting_geo_region: false # Default is true, will be overridden if pinterest__using_pin_promotion_targeting_report is set to False

Reviewer: Since these technically don't override each other anymore in the source, maybe this would make more sense:
```suggestion
    pinterest__using_pin_promotion_targeting_report: false # Default is true. Will disable `pinterest_ads__campaign_country_report` and `pinterest_ads__campaign_region_report` if false
    pinterest__using_targeting_geo: false # Default is true. Will disable `pinterest_ads__campaign_country_report` if false
    pinterest__using_targeting_geo_region: false # Default is true. Will disable `pinterest_ads__campaign_region_report` if false
```
